docs/components/connectors/out-of-the-box-connectors/email.md

```diff
@@ -0,0 +1,695 @@
+
+To use the **Email Connector**, you need to have an SMTP, POP3 or IMAP server available.
+Use Camunda secrets to avoid exposing your sensitive data as plain text. Follow our documentation
+on [managing secrets](/components/console/manage-clusters/manage-secrets.md) to learn more.
+
+# Authentication
+
+As of now, there is two different ways to authenticate to a mail server.
+
+## Simple Authentication
+
+It allows the user to connect to any SMTP, POP3 or IMAP server using the email address and the associated password.
+
+# POP3
+
+The Post Office Protocol version 3 (POP3) is an Internet standard protocol used by local email clients to retrieve
+emails from a remote server over a TCP/IP connection. POP3 allows users to download messages from their email server to
+their local computer, where they can be read, managed, or archived even without an internet connection. It operates on a
+simple download-and-delete model, meaning emails are typically removed from the server once they are retrieved.
+
+- `POP3 host`: the host url of the POP3 server.
+- `POP3 port`: the host port of the POP3 server.
+- `Cryptographic protocol`: It defines how the connection to the server should be secured, TLS, SSL or None. Default is
+  typically TLS.
+
+## List Emails
+
+### Parameters
+
+This feature allows users to fetch a list of emails from the `INBOX` folder with customizable sorting and limitation
+options.
+
+- `Max Emails to read`: Specify the maximum number of emails to retrieve. This parameter determines the cap on the
+  number
+  of emails the task will return.
+- `Sort emails by`: Choose the field by which to sort the emails. The currently supported sorting fields are:
+    - `Sent date`: Sorts emails by the date and time they were sent.
+    - `Size`: Sorts emails by the size of the email.
+- `Sort order`: Define the sort order using either:
+    - `ASC`: Ascending order, from the oldest or smallest value to the most recent or largest.
+    - `DESC`: Descending order, from the most recent or largest value to the oldest or smallest.
+
+### Sorting and Limiting Behavior
+
+Emails are initially sorted based on the specified sorting field and order. The list is then limited to the number of
+emails as defined by the Max Emails to read parameter. For example, if you sort by Sent date in descending order (DESC)
+with a limit of one email, the task will return the most recently sent email.
+
+### Response Structure
+
+The task returns a list of emails in JSON format. Each email object contains the following information:
+
+- `messageId`: A unique identifier for the email message.
+- `fromAddresses`: An array of email addresses representing the senders.
+- `subject`: The subject line of the email.
+- `size`: The size of the email in bytes.
+
+### Example Response
+
+Example of a returned JSON array:
+
+```json
+[
+  {
+    "messageId":"RandomId",
+    "fromAddresses":["msa@communication.microsoft.com"],
+    "subject":"Example",
+    "size":99865
```

Comment:
comment for the connector ifself, can we return `sizeInBytes` instead of `size`?